docs/docs/import.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+section: docs
+topic: import
+---
+
+Import and use a LitElement-based element in an HTML document, or in another element.
+
+**On this page:**
+
+* [Import and use your own element](#own)
+* [Import and use a third-party element](#thirdparty)
+
+### Import and use your own element
+
+In an HTML document:
+
+```html
+<head>
+  <script type="module" src="/path/to/my-element.js"></script>
+</head>
+<body>
+  <my-element></my-element>
+</body>
+```
+
+In another LitElement-based element:
+
+```js
+// Use relative paths for peer dependencies
+import './my-element.js';
+
+class MyOtherElement extends LitElement{
+  render(){
+    return html`
+      <my-element></my-element>
+    `;
+  }
+}
+customElements.define('my-other-element', MyOtherElement);
+```
+
+<a name="thirdparty">
+
+### Import and use a third-party element
+
+To work with an existing LitElement-based element made by a third party, refer to the third party documentation for instructions on how to install the element and use it in your project.
+
+Many elements are published on npm and can be installed from the command line:
+
+```bash
+cd my-project-folder
+npm install package-name --save
+```
+
+In an HTML document, an element published on npm can be imported from the `node_modules` folder:
+
+```html
+<head>
+  <script type="module" src="node_modules/package-name/existing-element.js"></script>
```

Comment:
I would suggest importing by name with inline script instead and let the tools will handle the resolution:

```html
<script type=module>
import 'package-name/existing-element.js';
</script>
```

This is what we did for PolymerElement demos (e.g. https://github.com/PolymerElements/iron-list/blob/master/demo/index.html#L20).